aderyn_core/src/report/markdown_printer.rs

```diff
@@ -218,11 +229,17 @@ impl MarkdownReportPrinter {
 
             writeln!(
                 writer,
-                "- Found in {} Line: {}\n\n\t```solidity\n\t{}\n\t```\nfile://{}\n\n\n\n",
+                "- Found in {} [Line: {}]({}#L{})\n\n\t```solidity\n\t{}\n\t```\n",
```

Comment:
`#L` here stands for line number. A colon followed by number doesn't work here. 

tldr;
The link is sent to the vs-code server running remotely to resolve to the file. And you can imagine in a URL you add # to indicate sort of an ID. When you use colon, it breaks URL (firstly because it's not URLencoded and 2nd - it implies you are trying to connect to a port of some sort)..That's why you get HostNotFound type of errors

Reply:
Makes sense, thank you for the explanation